src/components/common/CheckboxRadio/stories.tsx

```diff
@@ -14,11 +14,11 @@ export const argTypes = {
 	},
 	size: {
 		options: ['sm', 'md', 'lg'],
```

Comment:
I know you haven't touched this but can we use here that enum `SizingSmMdLg` please?